concepts/numbers/about.md

```diff
@@ -1,3 +1,107 @@
 # About
 
-TODO: add information on numbers concept
+There are two different kinds of numbers in JavaScript - numbers and bigints
```

Comment:
"big integers" might be written out, because it is defined in detail below.  Otherwise quoted, as "bigints" is not likely to be found in an English dictionary, and it is jargon.